src/components/body/body-row.component.ts

```diff
@@ -61,7 +61,12 @@ export class DataTableBodyRowComponent implements DoCheck {
 
   @Input() expanded: boolean;
   @Input() rowClass: any;
-  @Input() row: any;
+  public row: any;
+  @Input('row') set rowSetter(val: any) {
+    this.zone.run(() => {
```

Comment:
Wouldn't `cd.markForCheck` be better here?

Reply:
I don't know much about **ChangeDetectorRef** , I'll try that out, if works I'll change it.